src/components/Pool/PairsList/index.tsx

```diff
@@ -60,14 +60,16 @@ export default function PairsList({ aggregatedPairs, loading, filter, singleSide
   const [page, setPage] = useState(1)
   const responsiveItemsPerPage = useResponsiveItemsPerPage()
   const itemsPage = usePage(aggregatedPairs, responsiveItemsPerPage, page, 0)
-  const { address: swprAddress } = useSWPRToken()
+  const SWPRToken = useSWPRToken()
+  const swprAddress = SWPRToken?.address ?? undefined
```

Comment:
shouldn't this be `swaprAddress`?

Reply:
We are also using `swaprAddress` here.

https://github.com/levelkdev/swapr-dapp/pull/968/files#diff-cb9d5ea804236c5f52daa785bbf95c132da46bfa2c783e85e0f0c3cda545dbbaR66

Reply:
It's an opinionated debate. `SWPR` is the token symbol, so `swprAddress` makes sense or `swprTokenAddress`.

Reply:
`swprAddress` it's ok as long as we use always the same naming. 👍